build-system/tasks/update-packages.js

```diff
@@ -22,7 +22,12 @@ const log = require('fancy-log');
 const {exec, execOrDie, getStderr} = require('../exec');
 const {isTravisBuild} = require('../travis');
 
-const yarnExecutable = 'npx yarn';
+/**
+ * NOTE: executing yarn with --production=false prevents having
+ * NODE_ENV=production variable set which forces yarn to not install
+ * devDependencies. This usually breaks gulp for example.
+ */
+const yarnExecutable = 'npx yarn --production=false';
```

Comment:
After reading the `yarn install` [docs](https://yarnpkg.com/lang/en/docs/cli/install/), I realized that the `--production` flag is only used by `yarn install`. (It's true that simply running `yarn` defaults to `yarn install`.)

To restrict this change to just the case that matters, could I ask you to undo this change here, and make the following change here?

https://github.com/ampproject/amphtml/blob/b84350059273cec5d8e683b3bfae073cf356dcd1/build-system/tasks/update-packages.js#L184-L185

```
    log('Running', colors.cyan('yarn'), 'to update packages...');
    // NOTE: executing yarn with --production=false prevents having
    // NODE_ENV=production variable set which forces yarn to not install
    // devDependencies. This usually breaks gulp for example.
    execOrDie(yarnExecutable);
```